From 99e745d2117f884c41e51393111c5143507d0f51 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Sun, 11 Aug 2024 23:26:34 +0200
Subject: [PATCH] set fixed libs

---
 src/Makevars | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/Makevars b/src/Makevars
index 831c90f..852dd55 100644
--- a/src/Makevars
+++ b/src/Makevars
@@ -1,3 +1,2 @@
-PKG_CPPFLAGS = -I. -I../inst/include
-## Use the R_HOME indirection to support installations of multiple R version
-PKG_LIBS = `$(R_HOME)/bin/Rscript -e "RcppGSL:::LdFlags()"`
+PKG_LIBS = -lgsl -lgslcblas -lm
+PKG_CPPFLAGS = -I. -I../inst/include
